ote/src/clj/ote/services/associations.sql

```diff
@@ -0,0 +1,17 @@
+-- name: add-operator-service-association
```

Comment:
Is this in use anymore?

Reply:
It wasn't used anywhere so I removed it. Good point